app/assets/javascripts/testframework_config.js

```diff
@@ -13,27 +13,27 @@ function toggleSettings ( collapse_lnk ) {
   left_side = box.find('.settings_left_side');
   right_side = box.find('.settings_right_side');
 
-  max_marks = box.find('.maxmarks');
+  //max_marks = box.find('.maxmarks');
   desc = left_side.find('.desc');
   desc_box = desc.find('textarea');
 
   if( collapse_lnk.data('collapsed') ) {
     // Was collapsed. Need to expand.
     desc.nextAll('*').show();
-    max_marks.nextAll('*').show();
+    //max_marks.nextAll('*').show();
```

Comment:
I suppose you can delete these.

Reply:
That js file is not currently used, but I wanted to keep track of the original behavior in case we want to use it again.
Commenting means that if somebody puts it back as-is, at least it does not break. I'm open to leave it without the comments or even dump it altogether.

Reply:
Ah, got it. Please just dump this file. If we later want to re-introduce some of this functionality, we'll use libraries to do so.

Reply:
Well it turns out it's being used instead for minor things (setting the file name when clicking browse and greying the form when clicking delete). I'll dump the unused stuff.